Makefile

```diff
@@ -1,25 +1,35 @@
-PKG_CONFIG	= pkg-config
-WARNFLAGS	= -Wall -Werror
-PNGFLAGS	!= ${PKG_CONFIG} --cflags libpng
-REALCFLAGS	= ${CFLAGS} ${WARNFLAGS} ${PNGFLAGS} -Iinclude -g \
-		-std=c99 -D_POSIX_C_SOURCE=200809L
+# User-defined variables
+
+Q		:= @
+PREFIX		:= /usr/local
+bindir		:= ${PREFIX}/bin
+mandir		:= ${PREFIX}/man
+STRIP		:= -s
+BINMODE		:= 555
+MANMODE		:= 444
+
+# Other variables
+
+PKG_CONFIG	:= pkg-config
+PNGCFLAGS	:= `${PKG_CONFIG} --static --cflags libpng`
+PNGLDFLAGS	:= `${PKG_CONFIG} --static --libs-only-L libpng`
+PNGLDLIBS	:= `${PKG_CONFIG} --static --libs-only-l libpng`
```

Comment:
This is slower than the previous system, but it makes it work in all platforms I've tested.

I'm not sure about the static flag.